Bottom: db1711d77333c4af49851bef860eb92c73f5f70f
Top:    e83e7c8d2e5b7f53343b6082d1d22205630a4e4f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-30 15:57:24 +0800

Refresh of vhost_net-split-out-datacopy

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 33e6428..b2e1f2b 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -535,7 +535,6 @@ static void handle_tx_copy(struct vhost_net *net)
 	};
 	size_t len, total_len = 0;
 	int err;
-	size_t hdr_size;
 	struct socket *sock;
 	int sent_pkts = 0;
 
@@ -550,8 +549,6 @@ static void handle_tx_copy(struct vhost_net *net)
 	vhost_disable_notify(&net->dev, vq);
 	vhost_net_disable_vq(net, vq);
 
-	hdr_size = nvq->vhost_hlen;
-
 	for (;;) {
 		err = get_tx_bufs(net, nvq, &used, &msg, &out, &in, &len);
 		if (err == -ENOSPC) {
@@ -566,7 +563,6 @@ static void handle_tx_copy(struct vhost_net *net)
 			break;
 
 		total_len += len;
-
 		if (total_len < VHOST_NET_WEIGHT &&
 		    !vhost_vq_avail_empty(&net->dev, vq))
 			msg.msg_flags |= MSG_MORE;
